jetty-servlets/src/main/java/org/eclipse/jetty/servlets/CrossOriginFilter.java

```diff
@@ -501,11 +511,41 @@ private String commify(List<String> strings)
         return builder.toString();
     }
 
+    /** 
+     * creates a pattern to match any of a list of  string
+     * If the input list is ["foo, "bar"] it will generate
+     * a Pattern for the regex "(\\Efoo\\E|\\Ebar\\E) 
+     */
+    static Pattern patternize(final List<String> strings) 
+    {
+        final StringBuilder builder = new StringBuilder();
+        for (int i = 0; i < strings.size(); ++i)
+        {
+            final String string = strings.get(i);
+            if (string.trim().isEmpty()) break;
+            
+            if (builder.length() == 0) 
+                builder.append('(');
+            else
+                builder.append('|');
+            
+            builder.append(Pattern.quote(string));
```

Comment:
You.re trimming the `string` above but not here.

Reply:
Right! Fixed at `0ced641a`